split_settings/tools.py

```diff
@@ -108,10 +112,23 @@ def include(  # noqa: WPS210, WPS231, C901
 
             scope[_INCLUDED_FILE] = included_file
             with open(included_file, 'rb') as to_compile:
-                compiled_code = compile(  # noqa: WPS421
-                    to_compile.read(), included_file, 'exec',
-                )
-                exec(compiled_code, scope)  # noqa: S102, WPS421
+                if included_file.endswith('.py'):
+                    compiled_code = compile(  # noqa: WPS421
+                        to_compile.read(), included_file, 'exec',
+                    )
+                elif included_file.endswith('.pyc'):
```

Comment:
Please, move this to a new function.

Reply:
I've separated the logic into `_load_py` and `_load_pyc` functions in 4fff98f.